x/superfluid/keeper/stake.go

```diff
@@ -82,7 +82,7 @@ func (k Keeper) RefreshIntermediaryDelegationAmounts(ctx sdk.Context, accs []typ
 			adjustment := refreshedAmount.Sub(currentAmount)
 			err = k.mintOsmoTokensAndDelegate(ctx, adjustment, acc)
 			if err != nil {
-				ctx.Logger().Error("Error in forceUndelegateAndBurnOsmoTokens, state update reverted", err)
+				ctx.Logger().Error("Error in mintOsmoTokensAndDelegate, state update reverted", err)
 			}
```

Comment:
something weird I noticed here is that the log says "state update reverted", but I don't see any state update being reverted. Don't know this code enough to know if that is happening somewhere else though

Reply:
Yeah feels like this used to be a cache ctx apply func if no error or something